src/core/Akka.Tests/Actor/Dispatch/Bug2640Spec.cs

```diff
@@ -82,7 +82,6 @@ public async Task ForkJoinExecutorShouldShutdownUponActorSystemTermination()
 
             threads = ReceiveN(100).Cast<Thread>().GroupBy(x => x.ManagedThreadId)
                 .ToDictionary(x => x.Key, grouping => grouping.First());
-            threads.Count.Should().Be(4, "Expected 4 distinct threads in this example");
```

Comment:
We don't actually care about the amount of thread that is assigned to the test, we're testing if they're all terminated correctly.

Reply:
We did care about that early on - I haven't seen this spec act up in many years though. Why is it having problems now?

Reply:
This is a new addition from this PR, we don't have this test before

Reply:
See my explanation comment just before this

Reply:
Ah I see